src/components/gridList/demoDynamicTiles/index.html

```diff
@@ -1,5 +1,4 @@
 <div ng-app="gridListDemo" ng-controller="gridListDemoCtrl as vm" flex>
-  <md-button ng-click="vm.rotate()">Rotate</md-button>
```

Comment:
Yeah, didn't think it added much to the demo, but if you think otherwise I can add it back.